packages/forms/resources/js/components/date-time-picker.js

```diff
@@ -217,7 +217,10 @@ export default (Alpine) => {
                         return
                     }
 
-                    let date = this.getSelectedDate()
+                    let date = this.getSelectedDate() ?? dayjs().tz(timezone)
```

Comment:
I think this will cause unintended side effects, we should just convert `date.isAfter()` to `date?.isAfter()` etc with null checks in this method